frontend/client/__tests__/addmembermodal.react.test.js

```diff
@@ -0,0 +1,153 @@
+import { JSDOM } from 'jsdom'
+
+const dom = new JSDOM('<!doctype html><html><body></body></html>')
+global.window = dom
+global.document = dom.document
+
+import React from 'react'
+import Adapter from 'enzyme-adapter-react-16'
+import { configure, shallow, mount } from 'enzyme'
+import toJson from 'enzyme-to-json'
+import { rootStore } from '../src/store/model'
+import AddMemberModal from '../src/components/AddMemberModal'
+
+configure({ adapter: new Adapter() })
```

Comment:
Do we need this line? Shouldn't it have already been called in `test-setup.js`?

Reply:
See test manage members PR